aten/src/ATen/native/mkldnn/xpu/qlinear.cpp

```diff
@@ -0,0 +1,135 @@
+#include <torch/library.h>
+
+#include <ATen/native/mkldnn/xpu/detail/oneDNN.h>
+#include <c10/core/ScalarType.h>
+
+using namespace at::native::onednn;
+
+namespace at::native::xpu {
+
+Tensor q_linear_pointwise(
+    Tensor act, // int8 cpu tensor
```

Comment:
Do you mean `act` is a CPU tensor?

Reply:
oh, Appreciation for you reminding. It's my mistake putting these wrong comments. I removed this at my newest commit.